### DISCUSSION TITLE
Updating ECMA-335?

### DISCUSSION THREAD

**Discussion by IcedQuinn** [dotnet/runtime#93731] [General] [answered]
I see there is a log of changes to be made to the official ECMA spec. However, that spec has not been updated since before .NET Core 1.0. There has been production code with new metadata/tags (such as default interface methods) for ~eleven years now.

It might be about time to turn the TODO's to update the spec in to an updated spec :wink:

**Comment by tannergooding** [comment]
Please see the ECMA-335 augments file: https://github.com/dotnet/runtime/blob/main/docs/design/specs/Ecma-335-Augments.md

**Reply by IcedQuinn** [reply]
yes, i referred to that as "log of changes to be made."

**Comment by teo-tsirpanis** [comment, marked as answer]
Also per https://github.com/dotnet/runtime/pull/63659#issuecomment-1012654706, there are no plans for a seventh version of ECMA-335.

**Reply by IcedQuinn** [reply]
That is very disappointing. .NET still references being a specified platform, but it appears that pretense has long been thrown out the window. :woozy_face:
